tests/datetime/test_datetime.py

```diff
@@ -137,17 +152,70 @@ def test_needs_permission_for_no_timezones():
 @flaky(max_runs=3, min_passes=1)
 def test_bordering_on_a_leap_year():
     x = find(
-        datetimes(min_year=2002, max_year=2005),
+        datetimes(min_datetime=date(2003, 1, 1),
+                  max_datetime=date(2005, 12, 31)),
         lambda x: x.month == 2 and x.day == 29,
         settings=settings(database=None, max_examples=10 ** 7, timeout=-1)
     )
     assert x.year == 2004
 
 
+def test_can_draw_constrained_value():
+    start = datetime.min.replace(year=2001)
+    end = start + timedelta(microseconds=100)
+    drawn = datetimes(timezones=[],
```

Comment:
Yep.  `allow_naive=True` allows naive datetimes, but only setting `timezones` to an empty sequence *guarantees* them.  And I'm comparing the the naive min and max values, so it has to be a naive value.

(actually, I should also test drawing values from aware bounds)